src/scss/attendee/variables.scss

```diff
@@ -10,6 +10,8 @@ $text-darkmode: rgb(255, 255, 255);
 $panel-darkmode: rgb(0, 0, 0);
 $canvas-darkmode: rgb(50, 50, 50);
 
+$input-separator-color: rgba(130, 130, 130, 0.3);
```

Comment:
[NOTE] PR in progress for organizing these constants at https://github.com/sparkletown/sparkle/pull/2846

Reply:
replaced [235b7c0](https://github.com/sparkletown/sparkle/pull/2827/commits/235b7c012da13c3dcc07bbd66cbb7ad754f60dba)